### ISSUE TITLE
Cannot play strm files.

### ISSUE THREAD

**Issue by hidalgo67** [jurialmunkey/plugin.video.themoviedb.helper#444]
I’m a little bit confused how to set up a running system. Before Kodi 19 I used a Real-Debrid-plugin to have movies organized in my library and play the strm files. After the change to Kodi 19 it’s different. So I found this plugin to have strm files organized in my library and Seren to get a source. It works as expected but now I meet the tricky part. How to play a file (i.e. movie.strm) with this content?
`plugin://plugin.video.themoviedb.helper/?info=play&tmdb_id=363088&tmdb_type=movie&islocal=True`

I installed the corresponding player files from [https://github.com/a4k-openproject/json.openplayers](https://github.com/a4k-openproject/json.openplayers) but then what? I tried to choose a player in the settings but without success.

I do not have much developer skills, so please give me advice.

**Comment by drinfernoo** [comment]
Have you installed the proper player package? That repository has two branches 😉

I recommend installing from https://bit.ly/tmdbplayers, in TMDbHelper's settings. That will pull the most recent player files from the correct branch.

**Comment by hidalgo67** [comment]
> Have you installed the proper player package? That repository has two branches 😉
> 
> I recommend installing from https://bit.ly/tmdbplayers, in TMDbHelper's settings. That will pull the most recent player files from the correct branch.

Yes, I copied the seren files into the players folder. But I think, I have to configure the settings anyway and don’t have any clue what to do. Sometimes I see player file (Netflix and YouTube) in the select dialog, sometimes I don’t and don’t know why.
Any help to figure out, what to do?

**Comment by hidalgo67** [comment]
I don’t know, what happend, but now it works: I could choose a player from the list. That leads me to my next issue. First I decided to choose Play with Seren (Source Select). As I found out, that the first hit always works, I try to change that to Play with Seren, but this player does not show up. Although I can see and edit it in Configure players.
Is there any kind of cache or how to switch my player file?
